changelogs/unreleased/early-make-bootstrap-leader.md

```diff
@@ -0,0 +1,4 @@
+## feature/replication
```

Comment:
Nit: let's warn in the Doc request, that several `box.cfg.make_bootstrap_leader()` will lead to JOIN failure

Reply:
Ouch, missed in this push. Will do now.